inst/examples/03-formats.Rmd

```diff
@@ -218,9 +231,210 @@ favicon: "favicon.ico"
 
 A nice effect of setting `description` and `cover-image` is that when you share the link of your book on some social network websites such as Twitter, the link can be automatically expanded to a card with the cover image and description of the book.
 
+### Bootstrap 4 style {#bs4-book}
+
+The `bs4_book()` output format is built with Bootstrap (<https://getbootstrap.com>), using carefully crafted features to provide a clean reading experience whether you are on a phone, tablet, or desktop. You can read an example book here: <https://mastering-shiny.org/>
+
+```{r fig.align='center', echo=FALSE, include=identical(knitr:::pandoc_to(), 'html'), fig.link='https://mastering-shiny.org/', fig.alt="Screenshot of a bs4_book landing page to show the basic site layout"}
+knitr::include_graphics('images/bs4_book.png', dpi = NA)
+```
+
+This output format builds upon the Bootstrap CSS framework. Bootstrap is a library of reusable chunks of HTML, CSS, and JavaScript code that you can call on to modify your book without writing that code yourself. It is a free and open source project, and you can view the repository on GitHub: <https://github.com/twbs/bootstrap>. Bootstrap versions do get updated periodically; this format uses version 4 (<https://getbootstrap.com/docs/4.0/>).
+
+In addition to the basic **bookdown** components (Section \@ref(components)), the main features of `bs4_book` are:
+
+- Easy customisation of colours and fonts with
+  the **bslib** package (<https://pkgs.rstudio.com/bslib/>)
+
+- Built-in search (broken down by section) that helps readers quickly find what
+  they are looking for.
+
+- A sidebar containing a within-chapter table of contents that makes
+  navigation easy and helps provide context about your current position
+  within the chapter.
+
+- Thoughtful typography to make the contents as easy as possible to read,
+  regardless of the size of your device. A sticky header gets out of your
+  way when reading, but is easily accessible if you need it.
+
+- In-line footnotes mean you can read asides next to the next they refer
+  to. This theme is best paired with a reference style that generates
+  footnotes.
+
+- R syntax highlighting and autolinking by
+  the **downlit** package (<https://downlit.r-lib.org/>) is paired with a accessible
+  colour scheme designed by Alison Hill.
+  
+- Enhanced metadata for social sharing via platforms like Twitter, LinkedIn, and Facebook, so that each chapter shared will have a unique description, auto-generated based on that chapter's content.
+
+- Ability to configure links to a remote repository like GitHub or GitLab, allowing readers to easily view each chapter's source file or suggest an edit.
+
+The output format function in **bookdown** is `bs4_book()`. Here are its arguments:
```

Comment:
I think users will almost surely need `bookdown::bs4_book` instead of the function `bs4_book()`, so I suggest that we present the former directly:

```suggestion
The output format function is `bookdown::bs4_book`. Here are its arguments:
```